lib/libzfs/libzfs_util.c

```diff
@@ -59,6 +59,31 @@ libzfs_errno(libzfs_handle_t *hdl)
 }
 
 const char *
+libzfs_error_init(int error)
```

Comment:
I oped for this to be consistent with the existing `libzfs_error_action()` and `libzfs_error_description()` functions.  Consistency is good.